simple-golang-monitoring/cmd/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	hello_handler "simple-golang-monitoring/internal/modules/hello/handler"
+	internal_configs "simple-golang-monitoring/internal/pkg/configs"
+	pkg_http "simple-golang-monitoring/pkg/http"
+	pkg_otel "simple-golang-monitoring/pkg/otel"
+
+	"github.com/gin-gonic/gin"
+)
+
+func main() {
+	confLoader := internal_configs.NewConfigLoader()
+	configs := confLoader.LoadConfig()
+
+	app := pkg_http.NewHTTPServer(configs.AppEnv)
+
+	otel, err := pkg_otel.NewOpenTelemetryTracer(&configs.OtelAddress, configs.AppName, configs.AppEnv, "golang-standarization")
+	if err != nil {
+		panic(err)
+	}
+
+	app.Use(gin.Logger(), gin.Recovery())
```

Comment:
coba liat middleware in https://uptrace.dev/get/instrument/opentelemetry-gin.html#gin-instrumentation

dia bakal munculin tracer span yg ada http request, status code
bs langsung dipasang